daemon/containerd/fake_service_test.go

```diff
@@ -244,48 +243,3 @@ func (s *delayedStore) Update(ctx context.Context, info content.Info, fieldpaths
 	s.delay()
 	return s.store.Update(ctx, info, fieldpaths...)
 }
-
-type memoryLabelStore struct {
```

Comment:
Looks like this was a copy of an implementation in `distribution` - should we remove that one as well? https://github.com/moby/moby/blob/8601b22f5db511354d643a7722d11d33aa7ae13f/distribution/manifest_test.go#L46-L68